oak-blob-cloud/pom.xml

```diff
@@ -32,7 +32,7 @@
 
     <properties>
         <netty.version>4.1.17.Final</netty.version>
-        <aws.version>1.11.330</aws.version>
+        <aws.version>1.11.587</aws.version>
```

Comment:
Is this change to the AWS SDK version required for the proposed code change?
I am not opposed to upgrading our AWS SDK dependency version, but I would prefer that this be done in a separate commit if possible.